src/main/resources/feedbackQuestionTextEditFormTemplate.html

```diff
@@ -0,0 +1,16 @@
+<div>
+    <br>
+    <div class="row">
+        <div class="col-xs-11 question-recommended-length">
+            [Optional]
+            <span data-toggle="tooltip" data-placement="top"
+                title="the recommended length is shown to the respondent but not enforced">
+                Recommended length</span>
+            for the answer:
```

Comment:
Maybe change this to `for the response` so that the wording is consistent with the wording of the response length count shown in the submission edit page.